frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -92,6 +92,36 @@ export const BucketsItem: React.FC<BucketsItemProps> = React.memo(
   ),
 );
 
+export const BucketsTitle: React.FC<BucketsTitleProps> = ({
+  objectsCount,
+  link,
+  error,
+  children,
+}) => {
+  let objectsBody;
+  if (!objectsCount && !error) {
+    objectsBody = <div className="skeleton-text" />;
```

Comment:
`obLoaded` should be passed here, to check for `skeleton-text`, error alone is not enough. Because currently, if there is no error from Prometheus, and `objectsCount` is coming as undefined, the skeleton component will be shown by default, but it should be shown only till the till query is loading.

![Screenshot from 2020-05-11 16-27-56](https://user-images.githubusercontent.com/12200504/81554427-64021880-93a4-11ea-8012-1177e759ce3b.png)

Reply:
Aware of it.
Did not push changes, as we had discussions on rgw buckets.
Will update soon.

Reply:
@a2batic fixed in current push.